features/other_a/orphan_work.feature

```diff
@@ -144,7 +144,6 @@ Feature: Orphan work
 
     Given I am logged in as "keeper"
       And I post the work "Half-Orphaned"
-      And I wait 1 second
```

Comment:
Unnecessary now that the 1-second wait is baked in to the `I add the co-author "name" to the work "Title"` step, whee.